kaznet/apps/users/tests/test_serializers.py

```diff
@@ -288,3 +293,226 @@ def test_password_validate(self):
             str(serializer_instance.errors['password'][0]),
             'This password is too short. It must contain '
             'at least 8 characters.')
+
+    def test_gender_and_gender_display(self):
+        """
+        Test gender field
+        """
+        userprofile = mommy.make("auth.User").userprofile
+        serializer = UserProfileSerializer(userprofile)
+        userprofile.gender = 1
+        self.assertEqual(serializer.data["gender"], str(userprofile.gender))
+        self.assertEqual(
+            serializer.data["gender_display"], userprofile.gender_display)
+
+        userprofile.gender = 2
+        serializer = UserProfileSerializer(userprofile)
+        self.assertEqual(serializer.data["gender"], str(userprofile.gender))
+        self.assertEqual(
+            serializer.data["gender_display"], userprofile.gender_display)
+
+    @staticmethod
```

Comment:
As mentioned above, please try not to use static methods in test classes, in Django.